.travis/build.sh

```diff
@@ -6,7 +6,7 @@ PROJECT=$1
 SUBPROJECTS=$2
 
 # Set variables for use in build scripts
-export APPNAME="sils-$(basename $PROJECT)"
+export APPNAME="metatree-$(basename $PROJECT)"
```

Comment:
this will be resolved to metatree-fairspace-pluto and metatree-fairspace-saturn...